react/state.md

```diff
@@ -136,31 +173,69 @@ import React, { useState } from "react"
 
 const App = () => {
   const [count, setCount] = useState(0)
+
+  // the handleClick method is invoked when the div is clicked
   const handleClick = () => {
     setCount(count + 1)
   }
 
   return (
     <>
       <h1>Counter Application</h1>
-      <p>Counter: {count}</p>
-      <button onClick={handleClick}>Increment</button>
+      <div onClick={handleClick}>Count {count}</div>
     </>
   )
 }
 
 export default App
 ```
 
+### Styling in React
+
+Now that the basic counter functionality is working we can add some styling. To define the edges of the counter we can add an outline. The first step will be to add a `className` to the counter JSX div element.
+
+**src/App.js**
+
+```javascript
+<div onClick={handleClick} className="counter-box">
+  Count {count}
+</div>
+```
+
+All the stylings for our React application is going to live in the `App.css` file. If we look in the file there is content already there. This is from the boilerplate code that created the React page with the black background and blue React icon. Since we deleted all that boilerplate code from `App.js` we can delete the boilerplate code from `App.css` too.
```

Comment:
Grammar -> 'All the stylings for our React application **are** going to live...'